### PR TITLE
Bookstore - Initial components

### PR DESCRIPTION
**Pull request by jcmunav63**
## Description
This project will lay the foundations for your Bookstore website. You will create the user interface using React.

You will also set up client-side routing using [React Router V6](https://reactrouter.com/en/main).

## General requirements
- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used the correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

## HTML/CSS requirements
- Follow our list of the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

## JavaScript requirements
- Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

## Project requirements
- Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- Create a directory for your reusable components: /components.
- Install [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
- [Setup your router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of <App>
- Add 2 routes in your application, each should render a page component:
   - /
   - /categories
- Create components inside of the /components directory:
   - Contains individual book state, as well as a button to delete the book
   - Contains a list to render individual books
   - Contains a form to create new books
   - Contains a navigation to link to the 2 routes you created
 Note: Styling is NOT required at this state

### CODE REVIEW

File: src/components/Booklist.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                <button type="button" className="lightbutton">Comments</button>
+                <button type="button" className="lightbutton">Remove</button>
+                <button type="button" className="lightbutton">Edit</button>
+              </div>
+            </div>
+            <div className="div3">
+              <Button className="buttonbook">Update progress</Button>
+            </div>
+          </li>
+        ))}
```
Comment (codepantha) [inline src/components/Booklist.js]: - [ ] Please create a `<Book />` component to render the individual book items using the highlighted piece of code. The `<Book />` component should take the book data as a prop 👍 This has the added advantage of making your code cleaner and more maintainable

File: src/components/Booklist.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      </ul>
+      <hr />
+      <div className="divnewbook">
+        <h4 className="newbook">ADD NEW BOOK</h4>
+      </div>
```
Comment (codepantha) [inline src/components/Booklist.js]: - [ ] Please create a form component like`<NewBookForm />` that renders a form with inputs for the title and the author of the book. This is an important requirement 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @jcmunav63 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Set up book store project with basic components using CRA 📌 
✔️ No linter errors 🟢 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please add a MIT license to your project as it helps others know how to use your code legally 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — APPROVED** [review]
Hi @jcmunav63  , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**